src/app/components/Dropdown.tsx

```diff
@@ -58,6 +59,7 @@ const TabHiddenDropDown = styled(({toggle, children, className}: Props) => {
   const container = React.useRef<HTMLElement>(null);
 
   useOnClickOutside(container, open, () => setOpen(false));
```

Comment:
in my testing this `useOnClickOutside` doesn't appear to be necessary anymore, i wonder if there might be something you can click on that can't receive focus, that might require this extra handler.

if you can't think of a reason to keep it we might as well remove it